packages/types/src/spec.ts

```diff
@@ -15,3 +15,11 @@ export interface FoundSpec extends SpecFile {
   fileExtension: string
   specType: Cypress.CypressSpecType
 }
+
+export type SetSpecsFoundBySpecPattern = {
+  path: string
+  testingType: 'e2e' | 'component'
+  specPattern?: string | string[]
```

Comment:
Maybe we can grab these from the global `Cypress` types? Eg `Cypress.TestingType`, etc. They should all be defined there.